docs/installation.md

```diff
@@ -4,9 +4,11 @@ layout: documentation
 ---
 
 Our goal is to make it simple to add Error Prone checks to your existing Java
-compilation. Please note that Error Prone must be run on JDK 17 or newer. (It
+compilation. Please note that Error Prone must be run on JDK 21 or newer. (It
 can still be used to build Java 8 code by setting the appropriate `-source` /
-`-target` / `-bootclasspath` flags.)
+`-target` / `-bootclasspath` or `--release` flags.)\
+If you still have to build with an older JDK version you can use
+[older unmaintained Error Prone versions](#running-error-prone-on-earlier-jdk-versions).
```

Comment:
I hope this link to the section below is helpful and works. At least in the GitHub Markdown viewer it works, but I am not sure if that applies also to the tool you are using for documentation generation.